### PR TITLE
Refactor Code, Added Tag option for action element on carousel image

### PR DESCRIPTION
**Pull request by johnang-del**
Screen Shot 2019-10-03 at 1 13 41 PM

### CODE REVIEW

File: src/styles/_carousel.scss [hunk 1]
```diff
@@ -143,7 +143,7 @@
 		}
 	}
 
-	&__link,
+	&__Link,
```
Comment (pauvalenzuela) [inline src/styles/_carousel.scss]: use small letter for the css. Thanks.

File: src/components/Carousel/Carousel.js [hunk 2]
```diff
@@ -109,7 +98,7 @@ class Carousel extends Component {
 	};
 
 	render() {
-		const { images, thumbnails } = this.props;
+		const { images, thumbnails, action } = this.props;
```
Comment (pauvalenzuela) [inline src/components/Carousel/Carousel.js]: I think we're not using props 'action' here. We can remove this one.

File: src/styles/_carousel.scss (hunk 1 above)
Comment (johnang-del) [reply src/styles/_carousel.scss]: this has been address with latest commit

File: src/components/Carousel/Carousel.js (hunk 2 above)
Comment (johnang-del) [reply src/components/Carousel/Carousel.js]: this has been address with latest commit

**Pull request merged**